synapse/res/templates/sso_auth_bad_user.html

```diff
@@ -0,0 +1,18 @@
+<html>
+<head>
+    <title>Authentication Failed</title>
```

Comment:
Part of me wonders if this really needs to be a separate template or if we can use the `sso_error.html` template?

Reply:
I tried that to start with, but the phrasing of the sso_error.html template is very much geared to server-side or integration problems. This particular failure is really a user error, so the generic error wording didn't seem to fit.